Description: remove a test that fails because of QtScript bug
 The stacktrace is (obtained on ppc64el):
   #0  SLL_PopRange (end=<synthetic pointer>, start=<synthetic pointer>, N=<optimized out>, head=0x3fffb32004e0) at ../3rdparty/javascriptcore/JavaScriptCore/wtf/FastMalloc.cpp:742
   #1  PopRange (end=<synthetic pointer>, start=<synthetic pointer>, N=<optimized out>, this=0x3fffb32004e0) at ../3rdparty/javascriptcore/JavaScriptCore/wtf/FastMalloc.cpp:2083
   #2  ReleaseToCentralCache (N=192, cl=<optimized out>, this=0x3fffb3200478) at ../3rdparty/javascriptcore/JavaScriptCore/wtf/FastMalloc.cpp:2795
   #3  QTWTF::TCMalloc_ThreadCache::Cleanup (this=0x3fffb3200478) at ../3rdparty/javascriptcore/JavaScriptCore/wtf/FastMalloc.cpp:2745
   #4  0x00003fffb724c6d8 in QTWTF::TCMalloc_ThreadCache::DeleteCache (heap=0x3fffb3200478) at ../3rdparty/javascriptcore/JavaScriptCore/wtf/FastMalloc.cpp:3069
   #5  0x00003fffb6a48604 in __nptl_deallocate_tsd () at pthread_create.c:157
   #6  0x00003fffb6a489b8 in start_thread (arg=0x3fffb441f160) at pthread_create.c:322
   #7  0x00003fffb745f870 in clone () at ../sysdeps/unix/sysv/linux/powerpc/powerpc64/clone.S:104
 .
 QBS is going to be ported away from QtScript, but in the mean time remove this test
 so it does not block the build.
Author: Dmitry Shachnev <mitya57@debian.org>
Forwarded: not-needed
Last-Update: 2015-07-11

--- a/tests/auto/api/tst_api.cpp
+++ b/tests/auto/api/tst_api.cpp
@@ -304,9 +304,6 @@
     QTest::newRow("Q_OBJECT in header")
             << QString("moc-hpp")
             << relativeExecutableFilePath("moc_hpp");
-    QTest::newRow("Q_OBJECT in header, moc_XXX.cpp included")
-            << QString("moc-hpp-included")
-            << relativeExecutableFilePath("moc_hpp_included");
     QTest::newRow("app and lib with same source file")
             << QString("lib-same-source")
             << relativeExecutableFilePath("HelloWorldApp");
